frontend/core-ui/src/modules/notification/my-inbox/hooks/useArchiveNotifications.tsx

```
@@ -0,0 +1,69 @@
+    archive({
+      variables,
+    })
+      .then(() => {
+        toast({ title: 'Archived successfully' });
+
+        setSelectedNotifications([]);
+      })
+      .catch((error) =>
+        toast({ title: 'Something went error', description: error?.message }),
+      );
+    callback && callback();
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)